scripts/parser-tests/typescript/index.js

```diff
@@ -2,15 +2,12 @@ import path from "path";
 import fs from "fs";
 import { fileURLToPath } from "url";
 import TestRunner from "../utils/parser-test-runner.js";
-import ErrorCodes from "./error-codes.js";
```

Comment:
Why this change?

Reply:
This used to be used for diagnostic information generated by `typescript`, but was changed to detect the `.errors.txt` file in a PR half a year ago.
So now we don't need it anymore, but I didn't delete this file in case there is any need in the future.